### DISCUSSION TITLE
Pasting a repo URL in the mobile app should take you directly to the repo

### DISCUSSION THREAD

**Discussion by sshquack** [community/community#14002] [Mobile] [answered]
Currently when you paste a URL into Github mobile app, it shows useless results. See video below:

https://user-images.githubusercontent.com/5678459/161116645-2ffc399a-6928-42ae-b041-2dfd864ee16e.mov

Here is an example of how this should work. When you paste a reddit page URL, it takes you right to that page:

https://user-images.githubusercontent.com/5678459/161117198-4dbbce58-c933-47a7-9cb7-b9cec5f550f3.mov

**Comment by jacortinas** [comment, marked as answer]
Thank you for feedback, @sshquack! I think it's a great idea and something we can do with most URLs that support deep-linking. I've created an issue and we will prioritize this change for a future release.

**Reply by sshquack** [reply]
Awesome -- Looking forward to seeing this feature soon!

Might be related to https://github.com/github/feedback/discussions/14001
